### PR TITLE
update icasa.ts

### PR DESCRIPTION
**Pull request by DeanR97**
Added support for long press on iCasa ICZB-KPD14S & ICZB-KPD18S switches

### CODE REVIEW

File: src/devices/icasa.ts [hunk 1]
```diff
@@ -63,8 +63,8 @@ const definitions: Definition[] = [
         description: 'Zigbee 3.0 Keypad Pulse 4S',
         meta: {battery: {dontDividePercentage: true}},
         fromZigbee: [fz.command_recall, legacy.fz.scenes_recall_click, fz.command_on, legacy.fz.genOnOff_cmdOn, fz.command_off,
-            legacy.fz.genOnOff_cmdOff, fz.battery, legacy.fz.cmd_move_with_onoff, legacy.fz.cmd_stop_with_onoff],
-        exposes: [e.battery(), e.action(['recall_*', 'on', 'off', 'brightness_move_up', 'brightenss_move_down', 'brightness_stop'])],
+            legacy.fz.genOnOff_cmdOff, fz.battery, legacy.fz.cmd_move_with_onoff, legacy.fz.cmd_stop_with_onoff, fz.command_store],
+        exposes: [e.battery(), e.action(['recall_*', 'on', 'off', 'brightness_move_up', 'brightenss_move_down', 'brightness_stop', 'store_*'])],
```
Comment (Koenkk) [inline src/devices/icasa.ts:67]: Instead of the wildcard (`*`), can you provide the actual numbers?

Comment (DeanR97) [reply src/devices/icasa.ts]: done :)

**Comment by GingerAdonis** [comment]
Finally! Thank you! :)

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**